#### About the Bessant Lab

We devise and implement novel data science and AI solutions for biomedical research, particularly when proteomics data is involved. The lab is based in the [Digital Environment Research Institute](https://www.qmul.ac.uk/deri/) at [Queen Mary](http://www.qmul.ac.uk/), [University of London](https://london.ac.uk/) and led by [Prof Conrad Bessant](https://www.linkedin.com/in/conradbessant). We are also affiliated with Queen Mary's [School of Biological and Chemical Sciences](http://www.sbcs.qmul.ac.uk/) and the [The Alan Turing Institute](https://www.turing.ac.uk/).

#### Join us

*As a postdoc:* We welcome enquiries from postdocs interested in applying for funding to join our lab. A summary of currently available funding schemes can be found [by clicking here](https://www.qmul.ac.uk/sbcs/research/postdoctoral_fellowships/). Any enquiries regarding this should be emailed to Conrad Bessant [(c.bessant@qmul.ac.uk)](mailto:c.bessant@qmul.ac.uk) with your CV, **including "PDRA" in the subject line**.

*As a PhD student:* We host PhD students from the following UK-based doctoral training programmes: [LIDo](https://www.lido-dtp.ac.uk/), [AIDD](https://www.qmul.ac.uk/deri/ukri-aidd-doctoral-training-programme/) and [HDiP](https://www.qmul.ac.uk/wiph/study-with-us/postgraduate-research/wellcome-trust-phd-programme-in-science/). Other funded PhD studentship opportunities are available occasionally, typically for candidates from the UK. These will be advertised widely, e.g. on [findaphd.com](https://www.findaphd.com/phds/?Keywords=Bessant). We are also willing to consider applications from exceptional graduates funded by scholarships, or other means of self-funding, from any country.

*As an MSc student:* Our [MSc Bioinformatics programme](https://qmul.ac.uk/msc-bioinfo) is one of the most popular in Europe. Apply now for late September start.

**Sadly, we do not have the capacity to host internships or work experience placements.**

#### Selected journal articles: Check these out for a flavour of our research

Saihi, H., Bessant, C. and Alazawi, W., 2023 [Automated and reproducible cell identification in mass cytometry using neural networks](https://doi.org/10.1093/bib/bbad392). *Briefings in Bioinformatics*

Byrom, B., Bessant, C., Smeraldi, F., Abdollahyan, M., Bridges, Y., Chowdhury, M. and Tahsin, A., 2023. [Deriving meaningful aspects of health related to physical activity in chronic disease: concept elicitation using machine-learning-assisted coding of online patient conversations](https://www.valueinhealthjournal.com/article/S1098-3015(23)00050-5/fulltext). *Value in Health*.

Waller, K.J., Saihi, H., Li, W., Brindley, J.H., De Jong. A., Syn, W., Bessant, C., Alazawi, W., 2023 [Single-cell phenotypes of peripheral blood immune cells in early and late stages of non-alcoholic fatty liver disease](https://www.e-cmh.org/journal/view.php?doi=10.3350/cmh.2022.0205). *Clinical and Molecular Hepatology*.

Gomez, E.A., Colas, R.A., Souza, P.R., Hands, R., Lewis, M.J., Bessant, C., Pitzalis, C. and Dalli, J., 2020. [Blood pro-resolving mediators are linked with synovial pathology and are predictive of DMARD responsiveness in rheumatoid arthritis](https://www.nature.com/articles/s41467-020-19176-z). *Nature Communications*, 11(1), pp.1-13.

Gadaleta, E., Fourgoux, P., Pirró, S., Thorn, G.J., Nelan, R., Ironside, A., Rajeeve, V., Cutillas, P.R., Lobley, A.E., Wang, J., Gea, E., Ross-Adams, H., Bessant, C., Lemoine, N.R., Jones, L.J., & Chelala, C. (2020). [Characterization of four subtypes in morphologically normal tissue excised proximal and distal to breast cancer](https://doi.org/10.1038/s41523-020-00182-9). *npj Breast Cancer*, 6, article number: 38.

Hijazi, M., Smith, R., Rajeeve, V., Bessant, C. & Cutillas, P.R. (2020). [Reconstructing kinase network topologies from phosphoproteomics data reveals cancer-associated rewiring](https://dx.doi.org/10.1038/s41587-019-0391-9). *Nature Biotechnology*, 38(4), 493–502.

Saha, S., Matthews, D.A., Bessant, C. (2018). [High throughput discovery of protein variants using proteomics informed by transcriptomics](https://dx.doi.org/10.1093/nar/gky295). *Nucleic Acids Research*, 46(10), 4893–4902.

Pearce, O.M., Delaine-Smith, R.M., Maniati, E., Nichols, S., Wang, J., Böhm, S., ... & Balkwill, F.R. (2018). [Deconstruction of a metastatic tumor microenvironment reveals a common matrix response in human cancers](http://cancerdiscovery.aacrjournals.org/content/8/3/304). *Cancer Discovery*, 8(3), 304-319.

Chatzimichali, E.A., & Bessant, C. (2016). [Novel application of heuristic optimisation enables the creation and thorough evaluation of robust support vector machine ensembles for machine learning applications](https://link.springer.com/article/10.1007/s11306-015-0894-4). *Metabolomics*, 12(1), 16.

Evans, V. C., Barker, G., Heesom, K. J., Fan, J., Bessant, C., & Matthews, D. A. (2012). [De novo derivation of proteomes from transcriptomes for transcript and protein identification](https://www.nature.com/articles/nmeth.2227). *Nature Methods*, 9(12), 1207.

#### Books

[Proteome Informatics](http://pubs.rsc.org/en/content/ebook/978-1-78262-428-8) (Royal Society of Chemistry, 2016)

[Building Bioinformatics Solutions](https://global.oup.com/academic/product/building-bioinformatics-solutions-9780199658565) (Oxford University Press, 2014)

#### Contact

Email: [c.bessant@qmul.ac.uk](mailto:c.bessant@qmul.ac.uk)

To schedule a call with Conrad, [click here](https://calendly.com/bessant).

For other details see [Conrad's official QMUL profile page](http://www.sbcs.qmul.ac.uk/staff/conradbessant.html).

