#### About the Bessant Lab

We devise and implement novel data analysis solutions for biomedical research, particularly when proteomics data is involved. The lab is based in the [School of Biological and Chemical Sciences](http://www.sbcs.qmul.ac.uk/) at [Queen Mary](http://www.qmul.ac.uk/), [University of London](https://london.ac.uk/).

The lab is responsible for the [Galaxy Integrated Omics (GIO)](http://gio.sbcs.qmul.ac.uk) web-based platform for integrated analysis of transcriptomic and proteomic data, and [PITDB](http://pitdb.org) - a database of translated genomic elements identified from proteomics informed by transcriptomics (PIT) experiments. We also produced [Classyfire](https://cran.r-project.org/web/packages/classyfire/index.html), an R package for performing well validated machine learning experiments.  

#### Selected journal articles: Check these out for a flavour of our research

Saha, S., Matthews, D.A., Bessant, C. (2018). [High throughput discovery of protein variants using proteomics informed by transcriptomics](https://dx.doi.org/10.1093/nar/gky295). *Nucleic Acids Research*. In press.

Pearce, O.M., Delaine-Smith, R.M., Maniati, E., Nichols, S., Wang, J., Böhm, S., ... & Balkwill, F.R. (2018). [Deconstruction of a metastatic tumor microenvironment reveals a common matrix response in human cancers](http://cancerdiscovery.aacrjournals.org/content/8/3/304). *Cancer Discovery*, 8(3), 304-319.

Saha, S., Chatzimichali, E.A., Matthews, D.A., Bessant, C. (2017). [PITDB: a database of translated genomic elements](https://academic.oup.com/nar/article/46/D1/D1223/4372529). *Nucleic Acids Research* 46.D1, D1223-D1228.

Chatzimichali, E.A., & Bessant, C. (2016). [Novel application of heuristic optimisation enables the creation and thorough evaluation of robust support vector machine ensembles for machine learning applications](https://link.springer.com/article/10.1007/s11306-015-0894-4). *Metabolomics*, 12(1), 16.

Fan, J., Saha, S., Barker, G., Heesom, K.J., Ghali, F., Jones, A.R., ... & Bessant, C. (2015). [Galaxy integrated omics: web-based standards-compliant workflows for proteomics informed by transcriptomics](http://www.mcponline.org/content/14/11/3087.short). *Molecular & Cellular Proteomics*, 14(11), 3087-3093.

Evans, V. C., Barker, G., Heesom, K. J., Fan, J., Bessant, C., & Matthews, D. A. (2012). [De novo derivation of proteomes from transcriptomes for transcript and protein identification](https://www.nature.com/articles/nmeth.2227). *Nature Methods*, 9(12), 1207.

#### Books

[Proteome Informatics](http://pubs.rsc.org/en/content/ebook/978-1-78262-428-8) (Royal Society of Chemistry, 2016)

[Building Bioinformatics Solutions](https://global.oup.com/academic/product/building-bioinformatics-solutions-9780199658565) (Oxford University Press, 2014)

#### Join us

*As a postdoc:* We welcome equiries from postdocs seeking to apply for [Marie Skłodowska-Curie Action Individual Fellowships](https://ec.europa.eu/research/mariecurieactions/actions/individual-fellowships_en). Please email Conrad Bessant [(c.bessant@qmul.ac.uk)](mailto:c.bessant@qmul.ac.uk) with your CV, inluding MSCA in the subject line.

*As a PhD student:* We currently have a funded PhD studentship available, for a candidate from the UK or other EU country. For more information [see the advert](https://www.findaphd.com/search/ProjectDetails.aspx?PJID=97045).

*An an MSc student:* Our [MSc Bioinformatics programme](https://qmul.ac.uk/msc-bioinfo) is one of the most popular in Europe. Apply now for late September start.

#### Contact

Twitter: [@bezzlab](https://twitter.com/bezzlab)

Email: [c.bessant@qmul.ac.uk](mailto:c.bessant@qmul.ac.uk)
